track/__main__.py

```diff
@@ -48,31 +47,49 @@ def utc2localtime(utc):
 
 online = None
 last_offline = None
+
+printName = True
 while True:
-    if contact_id in ['me', 'self']:
+    try:
+        # File IO
+        dataFile = open('trackerLogs.txt', 'a+', encoding = 'utf-8')
```

Comment:
There is no need to re-open a file at each iteration. It could be opened outside the loop.